kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -855,6 +861,85 @@ object KyuubiConf {
       .intConf
       .createWithDefault(100)
 
+  val SERVER_STATE_STORE_CLASS: ConfigEntry[String] =
+    buildConf("kyuubi.server.state.store.class")
+      .doc("Class name for server state store.")
+      .version("1.6.0")
+      .stringConf
+      .createWithDefault("org.apache.kyuubi.server.statestore.jdbc.JDBCStateStore")
+
+  val SERVER_STATE_STORE_JDBC_DATABASE_TYPE: ConfigEntry[String] =
+    buildConf("kyuubi.server.state.store.jdbc.database.type")
```

Comment:
Have you considered making the config items related to the JDBC store locally? Not a global configuration item.

Considering if it's a pluggable mechanism, then this config belongs to a plugin-level configuration?

Reply:
For jdbc datasource properties, we support that user can specify kyuubi conf with prefix: kyuubi.server.state.store.jdbc.datasource.

Reply:
do you mean that, we can define a local jdbc property file?